### PR TITLE
pep8 bug fixed cli  config 

### PR DESCRIPTION
**Pull request by shitalmule04**
##### SUMMARY

 All bug fixed present in the  config  file

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

config
##### ANSIBLE VERSION

```
2.5
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/ec2_ami_copy.py [hunk 1]
```diff
@@ -1,236 +0,0 @@
-#!/usr/bin/python
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/amazon/ec2_ami_copy.py:1]: @shitalmule04 Any reason that you removed this entire file ?

File: test/sanity/pep8/legacy-files.txt [hunk 2]
```diff
@@ -6,7 +7,8 @@ lib/ansible/modules/cloud/amazon/_ec2_ami_search.py
 lib/ansible/modules/cloud/amazon/_ec2_remote_facts.py
 lib/ansible/modules/cloud/amazon/_ec2_vpc.py
 lib/ansible/modules/cloud/amazon/_ec2_vpc_dhcp_options.py
-lib/ansible/modules/cloud/amazon/aws_kms.py
+lib/ansible/modules/cloud/openstack/_os_server_actions.py
```
Comment (Akasurde) [inline test/sanity/pep8/legacy-files.txt]: Do not add file again in legacy-files.

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by Akasurde** [comment]
@shitalmule04 Use  `git pull --rebase upstream devel` on your branch in order to rebase your current branch.

File: lib/ansible/modules/cloud/amazon/ec2_ami_copy.py (hunk 1 above)
Comment (shitalmule04) [reply lib/ansible/modules/cloud/amazon/ec2_ami_copy.py:1]: @Akasurde  I still not made any changes in lib/ansible/modules/cloud/amazon/_ec2_ami_search.py, it added  my mistake. After that I tried to rebase branch so, this file is removed. Is it ok, if I commit cli/config.py and  amazon/ec2_ami_copy.py  at same time?

**Comment by dagwieers** [comment]
I tried to fix your PR through the Github web interface, but I cannot restore the file you removed. So I am going to close this and propose your changes in a new PR.

**Pull request closed without merging**